zeppelin-web-angular/src/app/pages/workspace/share/result/result.component.ts

```diff
@@ -198,6 +200,7 @@ export class NotebookParagraphResultComponent implements OnInit, AfterViewInit,
   }
 
   private handleVisualizationBundles(bundles: HeliumVisualizationBundle[]): void {
+    console.log('bundles', bundles);
```

Comment:
It looks like a console.log was accidentally left in.

Sorry for the late reply. If it’s not actually being used, I agree that gradually removing Helium visualization interface that relies on Angular's dynamic rendering makes sense. I fully agree with reducing Angular dependencies, and the direction you suggested sounds good to me.